Please check target existence before create it

Error:
```
...
[cmake] CMake Error at /home/wsl/vcpkg/scripts/buildsystems/vcpkg.cmake:649 (_add_library):
[cmake]   _add_library cannot create ALIAS target "exiv2lib" because another target
[cmake]   with the same name already exists.
[cmake] Call Stack (most recent call first):
[cmake]   out/build/debug-local-with-vcpkg/vcpkg_installed/x64-linux/share/exiv2/exiv2Config.cmake:81 (add_library)
[cmake]   /home/wsl/vcpkg/scripts/buildsystems/vcpkg.cmake:904 (_find_package)
...
```

It's easy to reproduce it, just add the following lines into a cmake project.

```
find_package(exiv2 CONFIG REQUIRED)
find_package(exiv2 CONFIG REQUIRED) # <-- error occurs
```

This is caused by this line (`cmake/exiv2Config.cmake.in`):
```
# compatibility with non-aliased users
add_library(exiv2lib ALIAS Exiv2::exiv2lib)
```

Please, fix it.

--- a/cmake/exiv2Config.cmake.in
+++ b/cmake/exiv2Config.cmake.in
@@ -58,4 +58,7 @@ include("${CMAKE_CURRENT_LIST_DIR}/exiv2Targets.cmake")
 check_required_components(exiv2)
 
 # compatibility with non-aliased users
-add_library(exiv2lib ALIAS Exiv2::exiv2lib)
+if(NOT TARGET exiv2lib)
+  add_library(exiv2lib ALIAS Exiv2::exiv2lib)
+endif()
+

